### PR TITLE
Add SEP-24 withdrawal flow docs

### PR DESCRIPTION
**Pull request by Ifropc**

### CODE REVIEW

File: docs/building-apps/wallet/sep24.mdx [hunk 1]
```diff
@@ -238,6 +236,95 @@ val transactions = sep24.getTransactionsForAsset(asset, token)
 
 </CodeExample>
 
+## Submitting Withdrawal Transfer
+
+Previously, we took a look on starting withdrawal flow. Now, let's take a look at full example working with the withdrawal.
```
Comment (JakeUrban) [inline docs/building-apps/wallet/sep24.mdx:241]:
```suggestion
Previously, we took a look at starting the withdrawal flow. Now, let's take a look at a full example.
```

File: docs/building-apps/wallet/sep24.mdx [hunk 2]
```diff
@@ -238,6 +236,95 @@ val transactions = sep24.getTransactionsForAsset(asset, token)
...
+
+<CodeExample>
+
+```kt
+val withdrawal = sep24.withdraw(asset, authToken = token)
+```
+
+</CodeExample>
+
+Next, open interactive url as an iframe:
```
Comment (JakeUrban) [inline docs/building-apps/wallet/sep24.mdx:253]: there are a bunch of options for opening the URL in mobile apps so I'm not sure if we should use the term iframe.

File: docs/building-apps/wallet/sep24.mdx [hunk 3]
```diff
@@ -238,6 +236,95 @@ val transactions = sep24.getTransactionsForAsset(asset, token)
...
+After that we need to wait until the anchor is ready to receive funds. To do so, we will be waiting until transaction reaches `pending_user_transfer_start` status
+
+<CodeExample>
+
+```kt
+var statusChange: StatusUpdateEvent
+
+// Wait for user input
+do {
+  statusChange = withdrawalWatcher.channel.receive()
```
Comment (JakeUrban) [inline docs/building-apps/wallet/sep24.mdx:273]: How do I get a `withdrawalWatcher` object?

File: docs/building-apps/wallet/sep24.mdx [hunk 4]
```diff
@@ -238,6 +236,95 @@ val transactions = sep24.getTransactionsForAsset(asset, token)
...
+} while (
+  ((statusChange as? StatusChange) ?: throw Exception("Channel unexpectedly closed"))
+    .status != TransactionStatus.PENDING_USER_TRANSFER_START
+)
+```
+
+</CodeExample>
+
+Next, sign and submit Stellar transfer:
```
Comment (JakeUrban) [inline docs/building-apps/wallet/sep24.mdx:282]:
```suggestion
Next, sign and submit the Stellar transfer:
```

File: docs/building-apps/wallet/sep24.mdx [hunk 5]
```diff
@@ -238,6 +236,95 @@ val transactions = sep24.getTransactionsForAsset(asset, token)
...
+val transfer = stellar.transaction(keypair).transferWithdrawalTransaction(anchorTransaction, asset).build()
+
+transfer.sign(keypair)
+
+stellar.submitTransaction(transfer)
+```
+
+</CodeExample>
+
+Where `keypair` is the SEP-10 authenticated account. If you want to transfer funds from a different address, refer to (Changing Stellar Transfer Account)[#changing-stellar-transfer-account] section.
```
Comment (JakeUrban) [inline docs/building-apps/wallet/sep24.mdx:297]:
```suggestion
Where `keypair` is the SEP-10 authenticated account. If you want to transfer funds from a different address, refer to [Changing Stellar Transfer Account](#changing-stellar-transfer-account) section.
```

File: docs/building-apps/wallet/sep24.mdx [hunk 6]
```diff
@@ -238,6 +236,95 @@ val transactions = sep24.getTransactionsForAsset(asset, token)
...
+
+Finally, let's track transaction status updates. In this example we simply check if transaction has been completed:
```
Comment (JakeUrban) [inline docs/building-apps/wallet/sep24.mdx:299]:
```suggestion
Finally, let's track transaction status updates. In this example we simply check if the transaction has been completed:
```

File: docs/building-apps/wallet/sep24.mdx (hunk 2 above)
Comment (Ifropc) [reply docs/building-apps/wallet/sep24.mdx:253]: What term should we use?

Comment (JakeUrban) [reply docs/building-apps/wallet/sep24.mdx:253]: Yes, something like that

**Review by JakeUrban — APPROVED** [review]

**Pull request merged**